src/lib/server/getProjectPluginConfigs.test.ts

```diff
@@ -0,0 +1,18 @@
+import { expect } from 'chai'
+import { getProjectPluginConfigs } from './getProjectPluginConfigs'
+
+describe('getProjectPluginConfigs()', () => {
+  it('create plugin config object with empty configuration for base path', () => {
+    const configs = [
```

Comment:
Why are multiple config files being passed and then collapsed to a single config object?